src/Subscriptions/resources/admin/components/SubscriptionDetailsPage/Tabs/Records/index.tsx

```diff
@@ -1,10 +1,29 @@
+import { useFormState } from 'react-hook-form';
+import { __ } from '@wordpress/i18n';
+import { AdminSectionsWrapper } from '@givewp/components/AdminDetailsPage/AdminSection';
+import { RecordsSlot } from '../../slots';
+import Notice from '@givewp/admin/components/Notices';
+
 /**
  * @unreleased
  */
-export default function SubscriptionDetailsPageRecordsTab() {
+export default function SubscriptionDetailsPageRecordsTab() {    
+    const { isDirty } = useFormState();
+
     return (
-        <div>
-            <h2>Records</h2>
-        </div>
+        <>
+            {isDirty && (
+                <div style={{ marginBottom: 'var(--givewp-spacing-4)' }}>
+                    <Notice
+                        type="info"                        
+                    >
```

Comment:
We could keep this in a single line as we only have one attribute here.

Reply:
Resolv in a5fb0c980152d5b606214ab1a7e53a627173e380